seqr/utils/elasticsearch/es_search.py

```diff
@@ -292,7 +294,7 @@ def _filter_by_genotype(self, inheritance_mode, inheritance_filter, quality_filt
                 self._index_searches[index].append(self._search)
 
     def _filter_compound_hets(self, quality_filters_by_family, annotations_secondary_search):
-        indices = set(self._indices)
+        indices = sorted(set(self._indices), reverse = True)
```

Comment:
This was turned into a set to improve the lookups for `index in indices`. If you are just going to immediately turn it back into a list theres no point in doing this

Reply:
The order of the indices is real import according to the codes from es_search.py line 208~211:
```
            for index in self._indices:
                family_samples_by_id = self.samples_by_family_index[index]
                affected_status = _get_family_affected_status(family_samples_by_id, inheritance_filter)
                self._family_individual_affected_status.update(affected_status)
```
The index comes after will overwrite the previous index's `affected_status` with the same family.

If the lookup performance is a concern, I could move the `sorted()` to a few lines after. It looks like below:
```
    def _filter_compound_hets(self, quality_filters_by_family, annotations_secondary_search):
        indices = set(self._indices)

        paired_index_families = defaultdict(dict)
        if len(indices) > 1:
            sv_indices = [
                index for index in self.indices_by_dataset_type[Sample.DATASET_TYPE_SV_CALLS] if index in indices
            ]
            variant_indices = [
                index for index in self.indices_by_dataset_type[Sample.DATASET_TYPE_VARIANT_CALLS] if index in indices
            ]
       <<more codes skipped>>

        seen_paired_indices = set()
        comp_het_q_by_index = {}
        for index in sorted(indices, reverse = True):
```

Reply:
I removed sorting here and changed back a `sorted()` in a line following.